foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -30,8 +30,16 @@ Finally, this is your first JavaScript program built from scratch, so don't forg
    console.log(playRound(playerSelection, computerSelection));
    ~~~
 
-6. Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-   1. You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
+6. Write a NEW function called `game()`. Call the `playRound` function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
+   1. Remember loops? This is a great opportunity to use one to play those five rounds:
+
+      ~~~javascript
+      for (let i = 1; i <= 5; i++) {
+         // your code here!
+      }
+      ~~~
+
+   1. Make sure there are no functions declared inside of the `game()` function.
```

Comment:
I actually think this may serve to confuse since the instructions earlier now make it clear one should only *call* the function, and there isn't language that would lend one to declare `playRound` inside of `game`.

Reply:
I checked couple of recent RPS projects and none of them had any functions declared inside of them. As such, I am going to delete this line.